pkg/lease/model.go

```diff
@@ -114,3 +116,36 @@ func (c StatusReason) StatusReasonPtr() *StatusReason {
 	v := c
 	return &v
 }
+
+// StringPtr returns a pointer to the input string value
+func StringPtr(c string) *string {
```

Comment:
Also, this generic method for take a string and returning a pointer might be better off in a generic package as opposed to the model package.

Reply:
This is unused function, so I removed it.